documentation/src/main/asciidoc/topics/proc_using_spring_cache_provider.adoc

```diff
@@ -1,10 +1,8 @@
-[id='adding-spring-cache_{context}']
-= Using {brandname} as a Spring cache provider
+[id='using-spring-cache-provider_{context}']
+= Using {brandname} as a Spring Cache provider
 
-To add caching support to your application, use the following annotations:
-
-*  adds entries to the cache.
-* `@CacheEvict` removes entries from the cache.
+After adding {brandname} dependencies, you can enable cache abstraction of Spring by adding the `@EnableCaching` annotation to your configuration classes.
+This allows you to start using the `@Cacheable` on the methods that you want to retrieve cached values from.
```

Comment:
Reading through this I realize we've got the steps in the procedure in the wrong order. Should be:

1. Enable cache annotations in your application context in one of the following ways:
2. Annotate methods with `@Cacheable` to cache return values.
3. Annotate methods with `@CacheEvict` to remove old entries from the cache.